documentation/src/main/asciidoc/topics/ref_cache_store_jdbc.adoc

```diff
@@ -0,0 +1,25 @@
+[id='jdbc_cache_store']
+= JDBC String-Based Cache Stores
+JDBC String-Based cache stores, `JdbcStringBasedStore`, use JDBC drivers to load and store values in the underlying database.
+
+`JdbcStringBasedStore` stores each entry in its own row in the table, instead
+of grouping multiple entries into each row, to increase throughput for
+concurrent loads. `JdbcStringBasedStore` also uses a pluggable bijection that
+maps each key to a `String` object. The `key-to-string-mapper` interface
+defines the bijection.
+
+{brandname} provides a default implementation, `DefaultTwoWayKey2StringMapper`,
+that handles primitive types.
+
+[NOTE]
+====
+* By default {brandname} shares are not stored, which means that all nodes in the cluster write to the underlying store on each update. If you want operations to write to the underlying database once only, you must configure JDBC store as shared.
+
+* `JdbcStringBasedStore` does not currently support segmentation but is planned
+for a future version of {brandname}.
+
+* If you use Apache Derby databases with `JdbcStringBasedStore`, you must set
+`dataColumnType` to `BLOB` as follows:
++
+`<data-column name="DATA_COLUMN" type="BLOB"/>`
+====
```

Comment:
I think we can remove this. It's not a popular database and we don't provide any details about other database column types.